src/test/java/it/pagopa/pn/bff/mocks/ConsentsMock.java

```diff
@@ -54,4 +54,13 @@ public List<BffTosPrivacyActionBody> acceptTosPrivacyBodyMock() {
 
         return bffTosPrivacyBody;
     }
+
+    public List<BffTosPrivacyActionBody> acceptPgTosPrivacyBodyMock() {
+        List<BffTosPrivacyActionBody> bffTosPrivacyBody = new ArrayList<>();
+
+        bffTosPrivacyBody.add(new BffTosPrivacyActionBody().action(BffTosPrivacyActionBody.ActionEnum.ACCEPT).version("1").type(it.pagopa.pn.bff.generated.openapi.server.v1.dto.ConsentType.TOS_SERCQ));
```

Comment:
this is wrong. for pg we don't have TOS_SERCQ or DATAPRIVACY, but only TOS_DEST_B2B

Reply:
fix it